pyo3-macros-backend/src/method.rs

```diff
@@ -207,12 +207,11 @@ impl SelfType {
             SelfType::TryFromPyCell(span) => {
```

Comment:
It might be nice to rename this `TryFromPyCell` variant to `TryFromBoundRef`.

Reply:
Good idea!